src/OnlineSales/DTOs/CustomerDtos.cs

```diff
@@ -0,0 +1,64 @@
+﻿// <copyright file="CustomerDtos.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+
+namespace OnlineSales.DTOs;
+
+public class CustomerCreateDto
+{
+    public string LastName { get; set; } = string.Empty;
+
+    public string FirstName { get; set; } = string.Empty;
+
+    [Required]
+    [EmailAddress]
+    public string Email { get; set; } = string.Empty;
+
+    public string CompanyName { get; set; } = string.Empty;
+
+    public string Address1 { get; set; } = string.Empty;
+
+    public string Address2 { get; set; } = string.Empty;
+
+    public string State { get; set; } = string.Empty;
+
+    public string Zip { get; set; } = string.Empty;
+
+    public string Location { get; set; } = string.Empty;
+
+    public string Phone { get; set; } = string.Empty;
+
+    public int Timezone { get; set; } = 0;
+
+    public string Culture { get; set; } = string.Empty;
+}
+
+public class CustomerUpdateDto
```

Comment:
@Exmirai, there is a problem with Dto which you have not fixed yet. Please implement Customer API integration tests to find it and fix. Have a look at BlogPostTests.cs